uk.co.compucorp.civicrm.appraisals/js/test/mocks/models/appraisal-cycle.js

```diff
@@ -37,204 +39,207 @@ define([
-                        { name: 'Awaiting HR Approval', count: 0 },
-                        { name: 'Complete', count: 100 }
-                    ]
-                },
-                {
-                    id: '42138',
-                    name: 'Appraisal Cycle 8',
-                    active: true,
-                    status: 'Status #2',
-                    type: 'Type #1',
-                    period: { start: '08/08/2014', end: '08/08/2015' },
-                    nextDue: { type: 'Self Appraisal', date: '08/08/2028' },
-                    appraisalsTotal: 100,
-                    completionPercentage: 15,
-                    appraisalsCountByStep: [
-                        { name: 'Self Appraisal', count: 40 },
-                        { name: 'Assigned to Manager', count: 22 },
-                        { name: 'Awaiting Grade', count: 12 },
-                        { name: 'Awaiting HR Approval', count: 11 },
-                        { name: 'Complete', count: 15 }
-                    ]
-                },
-                {
-                    id: '42139',
-                    name: 'Appraisal Cycle 9',
-                    active: true,
-                    status: 'Status #3',
-                    type: 'Type #1',
-                    period: { start: '09/09/2014', end: '09/09/2015' },
-                    nextDue: { type: 'Self Appraisal', date: '09/09/2029' },
-                    appraisalsTotal: 100,
-                    completionPercentage: 10,
-                    appraisalsCountByStep: [
-                        { name: 'Self Appraisal', count: 40 },
-                        { name: 'Assigned to Manager', count: 22 },
-                        { name: 'Awaiting Grade', count: 22 },
-                        { name: 'Awaiting HR Approval', count: 16 },
-                        { name: 'Complete', count: 10 }
-                    ]
-                },
-                {
-                    id: '421310',
-                    name: 'Appraisal Cycle 10',
-                    active: true,
-                    status: 'Status #1',
-                    type: 'Type #4',
-                    period: { start: '10/10/2014', end: '10/10/2015' },
-                    nextDue: { type: 'Self Appraisal', date: '10/10/2030' },
-                    appraisalsTotal: 100,
-                    completionPercentage: 2,
-                    appraisalsCountByStep: [
-                        { name: 'Self Appraisal', count: 40 },
-                        { name: 'Assigned to Manager', count: 24 },
-                        { name: 'Awaiting Grade', count: 22 },
-                        { name: 'Awaiting HR Approval', count: 11 },
-                        { name: 'Complete', count: 2 }
+            /**
+             * Mocked cycles
+             */
+            mockedCycles: function () {
+                return {
+                    total: 10,
+                    list: [
+                        {
```

Comment:
Right, forgot to clean things up!

The mocked cycles in `api.appraisals` were used before we could get the cycles list directly from the API. Once I implemented the real `api.appraisals.all()` method, i forgot to remove them

For `api.appraisals.activeCycles()` I still need to use a fake list since we don't have the `active` flag on the model, but definitely I don't need anymore all that detailed list